entry/render/buildRenderEntry.js

```diff
@@ -1,21 +1,3 @@
-import makeExtractor from '../makeExtractor';
+import render from './render';
 
-// __sku_alias__renderEntry is a webpack alias
-// pointing to the consuming apps render entry
-import render from '__sku_alias__renderEntry';
-
-export default async renderParams => {
-  const { SkuProvider, getBodyTags, getHeadTags } = makeExtractor(
-    renderParams.webpackStats,
-    renderParams.publicPath
-  );
-
-  const app = await render.renderApp({ ...renderParams, SkuProvider });
-
-  return await render.renderDocument({
-    ...renderParams,
-    app,
-    bodyTags: getBodyTags(),
-    headTags: getHeadTags()
-  });
-};
+export default async renderParams => render(renderParams);
```

Comment:
`await render`?

Reply:
Tbh this is just a passthrough function now so it doesn't even need to be async really.